src/Build/Resources/Strings.resx

```diff
@@ -463,6 +463,13 @@
     <value>MSB4187: Failed to receive a response from the task thread in the timeout period "{0}" ms. Shutting down.</value>
     <comment>{StrBegin="MSB4187: "}</comment>
   </data>
+  <data name="UnhandledMSBuildError" xml:space="preserve">
+    <value>"This is an unhandled exception in MSBuild -- PLEASE FILE AN ISSUE AT https://github.com/dotnet/msbuild/issues/new.
+    Message:
```

Comment:
Can we add a newline just before message? The current error looks like this:

```
uild/issues/new. [C:\Users\forgind\Desktop\Archives\Bug-specific\6460\1.csproj]    Message:
    Illegal characters in path.
    Stack:
       at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)
```
I didn't even notice the word "Message" at first. It'd be clearer if it looked like this:

```
uild/issues/new. [C:\Users\forgind\Desktop\Archives\Bug-specific\6460\1.csproj]    
		Message: Illegal characters in path.
    Stack:
       at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)
```